src/App.js

```diff
@@ -189,11 +192,24 @@ export class App extends React.Component {
                   <PrivateRoute exact path='/request-access/:id' component={RequestAccess} />
                 </div>
               </Switch>
-            {this.state.userSession.isAuthenticated
-              && <Footer />}
-          </div>
-        </BrowserRouter>
-      </UserContext.Provider>
+                {this.state.userSession.isAuthenticated
+                && <Footer />}
+            </div>
+          </BrowserRouter>
+          <ToastContainer
+            position="bottom-right"
```

Comment:
@Chris-Boe, if we decide to change the position from the bottom-right, it's as easy as  changing this attribute right here!